code/datums/diseases/advance/symptoms/viral.dm

```diff
@@ -65,22 +65,22 @@ Bonus
 //////////////////////////////////////
 */
 
-/datum/symptom/viralreverse		
+/datum/symptom/viralreverse
 
 	name = "Viral aggressive metabolism"
 	desc = "The virus sacrifices its long term survivability to nearly instantly fully spread inside a host. \
 	The virus will start at the last stage, but will eventually decay and die off by itself."
 	stealth = 1
 	resistance = -4
 	stage_speed = 3
-	transmittable = -3
+	transmittable = -2 //Buffed to make it maybe somewhat good
```

Comment:
keep this at -3 or else it can potentially be used for very nasty death plagues. if you wanna buff it, make it not vaccinate on cure

Reply:
Sorry chief this was requested for the bounty. I didn't pull this idea myself.

Reply:
iirc -3 was the minimum to not be antag fuel, which is why it was removed the first time. making it not vaccinate on cure lets you make virus "potions" and stuff

Reply:
I can look at that then.

Reply:
What are those death plauges? The level 9 death plague symptoms have - 2 transmission already. With the negative 2 from viral aggressive, if you use the best transmission symptoms you can only get to an 8 transmission, so it'd still only be blood transmission and not contagious.

Reply:
> iirc -3 was the minimum to not be antag fuel, which is why it was removed the first time. making it not vaccinate on cure lets you make virus "potions" and stuff

It does vaccinate on cure though.

Reply:
> keep this at -3 or else it can potentially be used for very nasty death plagues. if you wanna buff it, make it not vaccinate on cure

Show me the death plague this change enables and I'll give datboytim permission to remove that change. I paid for this PR, so I'm the one to convince.

Reply:
So I will be honest, not vaccinating sounds way more interesting than standard buffing symptom that kinda sucks.

It's hard to imagine what precisely could be done but man I'd rather go for something cool than "Oh boy, number go brrrr"

Reply:
I'll be completely honest. The reason for changing it is entirely to enable a trans 12 viral aggressive metabolism macrophage, so I could use macrophage to spread positive low transmission illnesses like exo loco used to do, without causing wide spread destruction from long term macrophage. If it enables a death plague we can sttle for non vaccinating, though non vaccinating might buff death plagues more then the + 1 transmission.

Reply:
I'll be completely honest. The reason for changing it is entirely to enable a trans 12 viral aggressive metabolism macrophage, so I could use macrophage to spread positive low transmission illnesses like exo loco used to do, without causing wide spread destruction from long term macrophage. If it enables a death plague we can settle for non vaccinating, though non vaccinating might buff death plagues more then the + 1 transmission.

Reply:
Not only is macrophage decently deadly in this form, and will attack once the disease is cured by VAM, it can be used to spread such a death plague with VAM on it- which with things like ARDS is incredibly tough to cure before dying. 

VAM is intended to, by design, be nearly impossible to spread on its own. It's for single-target diseases, not things you spread

Reply:
Macrophage isn't really that deadly, and not at all paired with regen coma and superficial healing (and the macrophage having a lowish stage speed). And VAM already works with a lot of different contagious illnesses as is. Autophage zombies, and anything wiith a transmission of at least 0 can be paired with it and spread via touch.

Reply:
It would only be a vector for use on LRP, admittedly.

Reply:
Look, macrophage is not intended as a vector for beneficials. DatBoi may end up making one such thing in the future if he continues keeping up my torch, but macrophage is still self-antag and against viro policy, at least if rules haven't gotten dumb. You cant do this as a nonantag anyways. VAM should not be compatable with transmission plagues that are worth anything. 

Also: said zombie virus doesnt spread itself unless something is terribly wrong with the code. Viromerol does not spread when the zombies attack.

Reply:
The rules have changed substantially for viro.

Reply:
The zombie virus is not acting as intended in that case, it does spread via attack.

Reply:
And what was or was not originally intended by the code is irrelevant.

Reply:
Zombies do spread via attack but the virus itself doesnt. 

And no, my original intent is not irrelevant if I revamped an entire department in a cohesive and balanced manner. Its a bunch of moving cogs that fit together for a reason. 

As it stands, i'd advise maintainers not to merge this pr in this state

Reply:
It is though, you left, now viro is rebalanced by others. By that logic no code should ever be changed except by its original creator. I paid for this pr, and with the attitude you are giving me, I will not have it changed. It will be merged this way or not merged.

Reply:
If you want it to be your pr, pay datboitim 15 bucks, and he can pay me back, and it'll be your pr. Until then, this isn't you pr Zesko don't act like you have final say. I was hearing you out and listening to your arguments. And I wouldn't say viro is balanced or cohesive just cause you made it, people already changed much of what you made proving it was neither.

Reply:
If you had provided this death plague you said was why the pr had to be changed, I would've changed it. As is, you are fumbling making up one reason after another when it really comes down to it not being your idea or original vision and you want to maintain control. Well, you shouldn't have quit in that case.

Reply:
Omigod you're actually getting pissy with me cuz i decided to come check the github just as a bunch of viro stuff was getting done?
I've checked.  There have been no merged viro prs since i left. Just because i am not here does not mean my opinion does not hold weight

No idea why you seem to hate the fuck out of me now. But given that shit hasn't changed, my opinion is still valid. Im not here to flamewar though. Just offer my two cents, cuz i know what im talking about

Anyways, i gotta get some sleep. Toodles~

Reply:
I don't hate you, you overstepped on this pr. And they have removed exo loco and made other changes since you left.

Reply:
Also: i just woke up im not designing any diseases, im about to go back to sleep

And no one has changed anything, unless you count austation's viro buff by lostman, which actively makes it overpowered

Viro is in a nice, cozy niche where it's powerful enough if you're good, okayish if not, but it has a hard power ceiling that makes it hard to game. If viro WERE gameable, it would very quickly annoy people due to its nature. And then it would get nerfed,  because of its nature.

Viro was nerfed in that way once. Twice, counting what i did to the major death plagues. Do you want that happening again?

Reply:
Anyways, brain stupid doesnt like coming back to old code shit yadda yadda dysphoria ill go to bed. Have fun.

Reply:
That has literally nothing to do with buffing VAMs transmission by 1. You are changing the subject. And you are one who said not to change it because "keep this at -3 or else it can potentially be used for very nasty death plagues. if you wanna buff it, make it not vaccinate on cure". All I asked was what that disease you are referencing is, and I showed the math for why that isn't true.

Reply:
Oh- exolocomotive was removed before i left. There were no other big changes iirc.  I had a breakdown before i could replace it. Like i said, i checked. It has to be replaced. But macrophage doesnt work in its niche.

Reply:
Anyways.  Paying for a pr wont get it automatically merged. Making a 12 transmission macrophage is already a potentially nasty death plague- it instantly starts spreading whatever nasty single target disease you can come up with, which can include another VAM disease

Reply:
I'm not saying its great in that niche, but we both agree that niche needs to be filled. This is the easiest way with minimum changes to the game. I'm not gonna pretend that macrophage spread for a good diseases is great. I just don't have anything else.

Reply:
So as for arguing about balance, I just want to add back an old ability we both know viro needed.

Reply:
God, dm me on discord or something. I dont wanna argue over github opened via my email in this shitty interface, spamming poor tim.

This is pointless

Reply:
We have talked on discord, and I am going with zesko's suggestion of making it not vaccinate on cure and not the transmission change.

Reply:
Too late for RIP Inbox.

As Zesko has said, I am carrying the torch. This means a xenolocomotive alternative is going to happen. Lee, I understand your buff was to enable a symptom to function similar to it, however, with a better vector coming in the pipeline, I feel it is safe to say that making macrophage a vector is not really the best approach for the purposes mentioned.

To my understanding, you both had come to an agreement as well. So we will call the issue of no-vax vs. transmission buff settled by both pipeline of a less physically aggressive vector and [Rule Of Cool](https://tvtropes.org/pmwiki/pmwiki.php/Main/RuleOfCool).

Code Resolution will happen soon.

Reply:
> If you want it to be your pr, pay datboitim 15 bucks, and he can pay me back, and it'll be your pr. Until then, this isn't you pr Zesko don't act like you have final say.

Pull your shit together dude. Zesko was voicing his concern in a respectful manner. I don't care who paid for the PR because it doesn't change anything, money does not add any weight to a balance decision.

Reply:
I am sorry, I overreacted.

Reply:
I was trying to understand his concerns and he just says, people should not merge this and I got upset, but what I said wasn't justified.

Reply:
I'll be resolving this to move forward with the agreed development path.